nncf/config/config.py

```diff
@@ -104,7 +104,9 @@ def _is_path_to_algorithm_name(path_parts: List[str]) -> bool:
     @staticmethod
     def validate(loaded_json):
         try:
-            jsonschema.validate(loaded_json, NNCFConfig.schema())
+            # jsonschema.validate(loaded_json, NNCFConfig.schema())
```

Comment:
Enable json validation

Reply:
Has not worked out so far

Reply:
Should be fixed here: https://github.com/vuiseng9/nncf/pull/13